sources/lib/api/gcp/context/_context.py

```diff
@@ -58,24 +58,35 @@ def project_id(self):
     return self._project_id
 
   @staticmethod
-  def default(project_id=None):
-    """Creates a default Context object.
+  def default(project_id=None, credentials=None):
```

Comment:
Would it be clearer if we have "default()" and "set_default()" functions to separate the getter and setter?